scripts/infra/cloud-instance.sh

```diff
@@ -474,6 +482,8 @@ Commands
         -n
             Name of the instance to terminate (default provided in config)
 
+    list - List all instances
```

Comment:
Should this say list all instance with specified name?

Reply:
I'm following the pattern for other command in the new revision. Lemme know if this is what you think of.

Reply:
It's a bit confusing because everything else is implicitly operating on 1 instance.  So it might not be clear what all instances you are listing still.  Would it be ok to state it as:

`List all applicable instances`

Reply:
Yeah; we could have `-a` to list *all* instances; but your suggestion is probably better one for the current behavior.